app/code/community/Razorpay/Payments/Helper/Data.php

```diff
@@ -32,6 +32,50 @@ public function isRazorpayEnabled()
         return Mage::getStoreConfigFlag(self::CONFIG_PATH_RAZORPAY_ENABLED);
     }
 
+    protected function verifyOrderAmount($razorpayOrderId, $order)
+    {
+        $url = $this->getRelativeUrl('order') . '/' . $razorpayOrderId;
+
+        $razorpayOrder = $this->sendRequest($url, 'GET');
+
+        $orderData = $this->getRazorpayOrderData($order);
+
+        $orderData['id'] = $razorpayOrderId;
+
+        $razorpayOrderKeys = array_keys($orderData);
+
+        foreach ($razorpayOrderKeys as $key)
+        {
+            if ($razorpayOrder[$key] !== $orderData[$key])
```

Comment:
Since iteration is over the order entity, you don't really need to separate `payment_capture` right? It can be part of `get..OrderData`?